### PR TITLE
Fix adding contact on Admin #586 

### PR DESCRIPTION
**Pull request by sergklukvin**
Fix validation ContactForm Component / add onBlur validation

### Checklist
- [ ] 🔽 My branch is up-to-date with "development" branch
- [ ] ✅ All tests passed locally
- [ ] ✨ My changes working with up-to-date front-end and back-end part locally, like charm
- [ ] 🔗 Link pull request to issue

### CODE REVIEW

File: src/components/forms/contacts-form/contacts-form.js [hunk 1]
```diff
@@ -100,24 +97,25 @@ const ContactsForm = ({ contactSaveHandler, initialValues }) => {
...
+      validateOnBlur: true,
+      onSubmit: (formValues) => {
+        if (
+          formValues.uaCartImage &&
+          formValues.enCartImage &&
+          typeof formValues.uaCartImage === typeof formValues.enCartImage
+        ) {
+          contactSaveHandler(formValues);
+        } else {
+          dispatch(setSnackBarSeverity('error'));
```
Comment (yuliiakucher) [inline src/components/forms/contacts-form/contacts-form.js:113]: please do not use magic string

File: src/components/forms/contacts-form/contacts-form.js [hunk 2]
```diff
@@ -100,24 +97,25 @@ const ContactsForm = ({ contactSaveHandler, initialValues }) => {
...
+          dispatch(setSnackBarMessage(SELECT_IMAGES_MESSAGE));
+          dispatch(setSnackBarStatus(true));
+        }
       }
-    }
-  });
+    });
 
   const inputs = [
```
Comment (yuliiakucher) [inline src/components/forms/contacts-form/contacts-form.js:120]: please move into utils

**Comment by yuliiakucher** [comment]
Please link issue

image

**Review by yuliiakucher — APPROVED** [review]

**Review by grebiniuk — APPROVED** [review]
Well done!

**Pull request merged**